### ISSUE TITLE
팀관리 수정

### ISSUE THREAD

**Issue by kwakchan** [limsangun/kick-kick#1]
<!DOCTYPE HTML>
<meta charset="utf-8">	
<html lang="ko">

<head>
	<title>팀 관리1</title>
	<meta name="viewport" content="width=device-width, initial-scale=1"> 
	<link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css" />
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
	<script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
	
	<style type="text/css">

		#table{
			text-align: center;
			padding: 10px;
			margin: 20px auto;
		}

		.team{
			border: 2px solid gray;
			padding: 10px;
			margin: 20px auto;
			width: 80%;
			height: 50%;
			
			font-size:20px;
			font-family: 'Nanum Gothic';
			color: gray;
			line-height: 50px;
			text-align: center;
			background: white;
			
			border-radius: 30px;
		}

		.circle{
			background-color:#0003;
			width:100px; height:100px;
			padding: 6px;
			border: 100px auto;
			margin: 10px auto;
			border-radius:75px;
			text-align:center;
			
			font-size:12px; 
			color: #black;
			vertical-align:middle;
			line-height:100px;
		}	
	</style>
</head>

<body>
	
	 <div data-role="page" id="bar">	
	 	
		<div data-role="header" data-position="fixed" id="header">
		</div>	
		
		
	    <div data-role="content" id="content">
	    	
	    	
			<div class="team">
				<p id="foo" style="text-align:center;">팀 소개</p>

				<table id="table">
					
					  <tr>
					    <td>
					    	<div class="circle">원 만들기1</div> 
					    </td>
					    <td>
					    	<div class="circle">원 만들기2</div>
					    </td>
					  </tr>

					  <tr>
					    <td>
					    	<div class="#">FC바르게살자</div>
					    </td>
					    <td>
					    	<div class="#">프로필</div>
					    </td>
					  </tr>
					
				</table>
			</div>

			
			<div class="team">
				<p id="foo" style="text-align:center;">팀 관리</p>

				<table id="table">
					
					  <tr>
					    <td>
					    	<div class="circle">포메이션</div> 
					    </td>
					    <td>
					    	<div class="circle">프로필</div>
					    </td>
					  </tr>
				</table>
			</div>
	    
	    </div> 

	    
	    <div data-role="footer" data-position="fixed" id="footer">   	
	    </div>	
	 </div>    

</body>

</html>
